__tests__/app.test.js

```diff
@@ -243,3 +243,22 @@ describe("DELETE /api/articles/:article_id", () => {
     expect(response.body.msg).toBe("Bad Request");
   });
 });
+
+describe("GET /api/users", () => {
+  it("200: responds with all of the users", async () => {
+    const response = await request(app).get("/api/users");
+    expect(response.status).toBe(200);
+    expect(response.body.users.length).toBe(4);
+    response.body.users.map((user) => {
```

Comment:
A forEach would be more appropriate here, we aren't needing to return a new array, just iterate through the response.